manual/manual/tutorials/coreexamples.etex

```diff
@@ -4,7 +4,7 @@
 
 This part of the manual is a tutorial introduction to the
 OCaml language. A good familiarity with programming in a conventional
-languages (say, Pascal or C) is assumed, but no prior exposure to
+languages (say, C or Javascript) is assumed, but no prior exposure to
 functional languages is required. The present chapter introduces the
```

Comment:
It pains me to see Javascript mentioned.  When we give examples of "conventional languages", can we choose some that don't completely suck?  Java might do.